src/operator/numpy/np_elemwise_broadcast_op.cc

```diff
@@ -144,5 +165,21 @@ MXNET_OPERATOR_REGISTER_NP_BINARY_SCALAR(_backward_npi_rcopysign_scalar)
 .set_attr<FCompute>("FCompute<cpu>",
                     BinaryScalarOp::Backward<cpu, mshadow_op::rcopysign_grad>);
 
+MXNET_OPERATOR_REGISTER_NP_BINARY_SCALAR(_npi_left_shift_scalar)
+.set_attr<FCompute>("FCompute<cpu>", BitScalarCompute<cpu, mshadow_op::left_shift>)
+.set_attr<nnvm::FGradient>("FGradient", MakeZeroGradNodes);
```

Comment:
Why the gradient is zero? as operator described, this operation is equivalent to multiplying x1 by 2**x2. So it should have same gradient as multiplying by 2.

Reply:
it's right, i wll add backward

Reply:
after talk with @reminisce , we think now, it should not has backward, because for left_shift(x1, x2) , x2's backward is x1 << x2 * log(2) , log(2) will be zero, we think we should provide backwrad with problem, if you want to use backward, you shoule use x1 * exp(2, x2). in order to use hybridize, i provide zero gradient,  in future, we will fix this problem with other same problem op.

Reply:
I agree with removing backward implementation for now. Please remove `FGradient` registration, because `MakeZeroGradNodes` will provide 0 gradient, which is unexpected.